osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -7,24 +7,33 @@
 using osu.Framework.Graphics;
 using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Objects.Drawables;
-using osu.Game.Rulesets.Objects.Types;
 using osu.Game.Rulesets.Osu.Objects;
 using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Configuration;
+using osu.Framework.Configuration;
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects
+    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects, IReadFromConfig
     {
         public override string Description => @"Play with no approach circles and fading notes for a slight score advantage.";
         public override double ScoreMultiplier => 1.06;
-
         private const double fade_in_duration_multiplier = 0.4;
         private const double fade_out_duration_multiplier = 0.3;
+        private Bindable<bool> increaseFirstObjectVisibility;

```

Comment:
you may create object and set to false by default

Reply:
Can we change the name to `firstObjectVisibility` (same with the setting) as it's a "we show (or not)" question.